functions/go/fix/go.mod

```diff
@@ -3,8 +3,9 @@ module github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/fix
 go 1.16
 
 require (
+	github.com/GoogleContainerTools/kpt-functions-sdk/go v0.0.0-20210810054117-e438c46aea31
 	github.com/stretchr/testify v1.7.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 	k8s.io/kube-openapi v0.0.0-20210421082810-95288971da7e
-	sigs.k8s.io/kustomize/kyaml v0.10.21
+	sigs.k8s.io/kustomize/kyaml v0.11.0
```

Comment:
The SDK repo also requires kayml. golang can only resolve it to v0.11.0.
Due to formatting change, this can be a breaking change for fix function.

Reply:
If the sdk requires kyaml, isn't it the same problem here ?